### PR TITLE
First draft of storyboard/scene lesson

### PR DESCRIPTION
**Pull request by mdippery**
cc: @AnnJohn @JimCampagno

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,17 +13,69 @@
...
+
+These two components, the editor and Interface Builder, are both parts of _Xcode_, and together, allow you to easily and rapidly develop both iOS and OS X graphical programs. The great thing about Xcode is that it allows you to lay out your program's interface using graphical tools, then easily connect the graphical components of your program to Swift code that makes them work.
+
+In this lesson, you'll get a quick introductory tour to Interface Builder. Start that tour by creating an Xcode project for a new iOS program. For this lesson, you can just use Xcode's Single View Application template. After creating this project, you'll be greeted with Xcode's standard editing window:
+
+![Xcode](.images/xcode01.png)
+
+Take a look at the file listing on the left-hand side of the window. You should see two `.storyboard' files: `Main.storyboard` and `LaunchScreen.storyboard`.
+
```
Comment (JimCampagno) [inline README.md:32]: Reminder to myself to edit the storyboard formatting word thingy.

File: README.md [hunk 2]
```diff
@@ -14,17 +13,69 @@
...
+You should also notice that on the left hand side of the Interface Builder window, next to the file listing, is a list of items. This area is the _object listing_. The topmost item is labeled **View Controller Scene**. Click once on this item. You should notice that the big blank view controller rectangle in the editing window is highlighting. **View Controller Scene** in the object listing corresponds to that view controller rectangle in the editing portion.
+
+![View Controller](.images/xcode04.png)
+
+### Scenes
+
+Have you ever seen a storyboard for a movie? It consists of drawings for key scenes of a movie, and shows how the story flows and what the movie will look like at each scene. iOS development uses a similar model to lay out graphical interfaces. An iOS app consists of one or more _screens_ which display text, images, icons—anything you want to show to the user. You move between these screens using controls like buttons.
+
```
Comment (JimCampagno) [inline README.md:56]: One thing to add here could be referencing something like the Contacts app people are familiar with. You have your list of contacts on one screen and by tapping on any individual within that list, we are presented with another screen.

I don't feel too strongly about that, so disregard if you think it doesn't add anything by stating something like that.

File: README.md [hunk 3]
```diff
@@ -14,17 +13,69 @@
...
+You store the layout of these different screens in the storyboard file. In keeping with the movie analogy, each screen is referred to as a _scene_. Generally, each scene has its own view controller. These screens are connected to each other in different ways, allowing you to easily move between them. The overall design and interaction of these screens is visible in the storyboard file, which allows you to have a high-level overview of the flow of your application.
+
+No better way to understand this than to see it in action!
+
```
Comment (JimCampagno) [inline README.md:62]: 👍

File: README.md (hunk 2 above)
Comment (mdippery) [inline README.md:56]: Added that.

**Comment by JimCampagno** [comment]
@mdippery Thanks for making the change. :)

**Pull request merged**